packages/gettext/gettext.0.3.6/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "1.2"
+maintainer: "Sylvain Le Gall <sylvain+ocaml@le-gall.net>"
+authors: [ "Sylvain Le Gall" ]
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "https://github.com/gildor478/ocaml-gettext"
+dev-repo: "https://github.com/gildor478/ocaml-gettext.git"
+bug-reports: "https://github.com/gildor478/ocaml-gettext/issues"
+build: [
+  [
+    "./configure"
```

Comment:
I think this is failing because the `configure` script is not included in the tarball.